benchmarks/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# benchmark-level CMakeLists.txt
+
+# For every *.cpp-file (every file corresponding to an executable) ...
+foreach(BENCH_SOURCE ${PROJECT_BENCH_SOURCE_FILES})
+
+
+    link_libraries("-lbenchmark")
```

Comment:
I feel like this line should be in the corresponding `cmake/ConfigureBenchmarks.cmake`-file.